charts/karpenter/values.yaml

```diff
@@ -27,7 +27,7 @@ serviceMonitor:
   enabled: false
   # -- Additional labels for the ServiceMonitor.
   additionalLabels: {}
-  # -- Endpoint configuration for the ServiceMonitor.
+  # -- Configuration on `http-metrics` endpoint for the ServiceMonitor. Not to be used to add additional endpoints 
```

Comment:
nit: Consider adding the second statement to a new line. It may also be useful to be more verbose about what could be configured here or pointing to the Prometheus operator documentation for the schema